Explorer/Assets/DCL/PerformanceAndDiagnostics/Diagnostics/SystemInfoUtils.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Text;
+using UnityEngine.Device;
+
+namespace DCL.Diagnostics
+{
+    public static class SystemInfoUtils
+    {
+        public static void Log()
+        {
+            var stringBuilder = new StringBuilder();
+
+            stringBuilder.Clear();
+            AppendHeader(ref stringBuilder, "DEVICE INFORMATION");
+
+            // Device & OS
+            stringBuilder.AppendFormat("Device Model: {0}\n", SystemInfo.deviceModel);
+            stringBuilder.AppendFormat("Device Name: {0}\n", SystemInfo.deviceName);
+            stringBuilder.AppendFormat("Operating System: {0}\n", SystemInfo.operatingSystem);
+            stringBuilder.AppendFormat("System Language: {0}\n", Application.systemLanguage);
+            stringBuilder.AppendFormat("Device Type: {0}\n", SystemInfo.deviceType);
+            stringBuilder.AppendFormat("Device Unique ID: {0}\n\n", SystemInfo.deviceUniqueIdentifier);
+
+            // CPU & Memory
+            AppendHeader(ref stringBuilder, "HARDWARE");
+            stringBuilder.AppendFormat("Processor Type: {0}\n", SystemInfo.processorType);
+            stringBuilder.AppendFormat("Processor Count: {0}\n", SystemInfo.processorCount);
+            stringBuilder.AppendFormat("Processor Frequency: {0} MHz\n", SystemInfo.processorFrequency);
+            stringBuilder.AppendFormat("System Memory Size: {0} MB\n\n", SystemInfo.systemMemorySize);
+
+            // Graphics
+            AppendHeader(ref stringBuilder, "GRAPHICS");
+            stringBuilder.AppendFormat("Graphics Device Name: {0}\n", SystemInfo.graphicsDeviceName);
+            stringBuilder.AppendFormat("Graphics Device Type: {0}\n", SystemInfo.graphicsDeviceType);
+            stringBuilder.AppendFormat("Graphics Memory Size: {0} MB\n", SystemInfo.graphicsMemorySize);
+            stringBuilder.AppendFormat("Graphics Device Version: {0}\n", SystemInfo.graphicsDeviceVersion);
+            stringBuilder.AppendFormat("Max Texture Size: {0}\n", SystemInfo.maxTextureSize);
+            stringBuilder.AppendFormat("Supports Ray Tracing: {0}\n\n", SystemInfo.supportsRayTracing);
+
+            // Unity & Application
+            AppendHeader(ref stringBuilder, "APPLICATION");
+            stringBuilder.AppendFormat("Unity Version: {0}\n", Application.unityVersion);
+            stringBuilder.AppendFormat("Company Name: {0}\n", Application.companyName);
+            stringBuilder.AppendFormat("Product Name: {0}\n", Application.productName);
+            stringBuilder.AppendFormat("Version: {0}\n", Application.version);
```

Comment:
The class [DCLVersion ](https://github.com/decentraland/unity-explorer/blob/f8ef3a1fc936dae037bf3cdef5ad81066fe7a088/Explorer/Assets/Scripts/Global/Versioning/DCLVersion.cs#L10) was introduced recentrly. Maybe you can use it here to avoid divergence